provider/import.go

```diff
@@ -80,16 +80,13 @@ func Import(ctx context.Context, p Provider, hcl, tfstate writer.Writer, f *filt
 				err = util.RetryDefault(func() error { return r.Read(f) })
 				if err != nil {
 					cause := errors.Cause(err)
+
 					// All those ignored errors are types of errors to identify that something happend, but not that something went wrong
 					// some of them are to skip the resource and others just informative
-					if cause != errcode.ErrProviderResourceNotRead && cause != errcode.ErrProviderResourceDoNotMatchTag && cause != errcode.ErrProviderResourceAutogenerated {
-						return errors.Wrapf(err, "could not read resource %s: ", r.Type())
-					}
-					if cause == errcode.ErrProviderResourceNotRead || cause == errcode.ErrProviderResourceAutogenerated {
-						// As the resource could not be Read, meaning an ID == ""
-						// we'll continue to the next resource
-						continue
-					}
+					// we'll continue to the next resource
+					fmt.Fprintf(out, "\ncould not read resource: %s.%s\n", re.Type(), re.ID())
+					logger.Log("error", cause)
```

Comment:
Didn't we say to not `fmt.Fprintf` the error and only `logger.Log` them? Also add more info on the `.Log` not just `error`, add more context, check others.

Reply:
Ok I was thinking to display kind of "warning" something have not been imported and put the details of the error in the log. Should we really "hide" everything that have not been imported ?

Reply:
IIRC you mentioned to see those only when debugging, that's with `-v` which is with the `logger.Log`.